src-docs/spec/README.md

```diff
@@ -1,99 +1,12 @@
-the following basic information MUST be read from it: 
-
-```toml
-name = "My App Name"
-source_code_url = "https://example.org/orga/repo"
-```
-
-- `name` - The name of the webxdc app.
-  If no name is set or if there is no manifest, the filename is used as the webxdc name.
-
-- `source_code_url` - Optional URL where the source code of the webxdc and maybe other information can be found.
-  Messenger implementors may make the url accessible via a "Help" menu in the webxdc window.
-
-
-### Icon Files 
-
-If the ZIP-root contains an `icon.png` or `icon.jpg`,
-these files are used as the icon for the webxdc.
-The icon should be a square at reasonable width/height,
-usually between 128 x 128 and 512 x 512 pixel.
-Round corners, circle cut out etc. will be added by the implementations as needed;
-do not add borders or shapes to the icon therefore.
-If no icon is set, a default icon will be used.
-
-## Messenger Implementation
-
-### Webview Constraints for Running Apps 
-
-When starting a web view for a webxdc app to run, messenger implementors:
-
-- MUST deny all forms of internet access. If you don't do this
-  unsuspecting users may leak data of their private interactions to outside third parties. 
-  You do not need to offer "privacy" or "cookie" consent screens as 
-  there is no way a WebXDC app can implicitly transfer user data to the internet. 
-
-- MUST allow unrestricted use of DOM storage (local storage, indexed db and co), 
-  but make sure it is scoped to each webxdc app so they can not delete or modify 
-  the data of other webxdc content.
-
-- MUST inject `webxdc.js` and implement the
-  [Webxdc API](#webxdc-api) so that messages are relayed and shown in chats. 
-
-- MUST make sure the standard JavaScript API works as described at
-  [Other APIs and Tags Usage Hints](/tips_and_tricks/compatibility.md#other-apis-and-tags-usage-hints).
-
-### UI Interactions in Chats
-
-- Text from `update.info` SHOULD be shown in the chats
-  and tapping them should jump to their webxdc message
-
-- The most recent text from `update.document` and `update.summary` SHOULD be shown inside the webxdc message,
-  together with name and icon.  
-  Only one line of text SHOULD be shown and truncation is fine
-  as webxdc devs SHOULD NOT be encouraged to send long texts here.
-
-- A "Start" button SHOULD run the webxdc app.
-
-### Example Messenger Implementations
-
-- [Android using Java](https://github.com/deltachat/deltachat-android/blob/master/src/org/thoughtcrime/securesms/WebxdcActivity.java)
-
-- [iOS using Swift](https://github.com/deltachat/deltachat-ios/blob/master/deltachat-ios/Controller/WebxdcViewController.swift)
-
-- [Desktop using TypeScript](https://github.com/deltachat/deltachat-desktop/blob/786b7514d69ffb723bbe6e706494852a2641bfcd/src/main/deltachat/webxdc.ts)
+- [`.xdc` container file format](format.md) describes zip-file format
+  with manifest and icon files. 
 
+- [Webxdc API](api.md) specifies a minimal stable API that all webxdc
```

Comment:
```suggestion
- [Javascript API](api.md) specifies a minimal stable API that all webxdc
```

The headline of the API section (2.2) is "Javascript API" not "Webxdc API".
(Note: [according to wikipedia](https://de.wikipedia.org/wiki/JavaScript), it's capitalized `JavaScript` - we don't need to respect that of course.)